src/pages/en/guides/debugging.md

```diff
@@ -4,4 +4,64 @@ title: Debugging
 description: Debug in Astro using the Debug component
 ---
 
-Astro runs on the server and logs directly to your terminal, so it can be difficult to debug values from Astro. Astro’s built-in `<Debug>` component can help you inspect values inside your files on the clientside. Read more about the [built-in Debug Component](/en/reference/builtin-components#debug-).
+Astro provides several different tools to help you debug your code easier and faster.
+
+## Debugging with `console.log()`
+
+`console.log()` is a simple-but-popular method of debugging your Astro code, especially in the component front matter.  
+
+
+```astro
+---
+const sum = (a, b) => a + b;
+
+// Example: Outputs "4" to the CLI terminal
+console.log(sum(2, 2));
+---
+```
+
+A `console.log()` statement in Astro frontmatter will always output to the **terminal** running the Astro CLI. This is because Astro runs on the server, and never in the browser.
+
+## Debugging JS Scripts
```

Comment:
Should this and the following heading be `<h3>` to nest under “Debugging with `console.log()`”?

```suggestion
### Debugging JS Scripts
```

The headings don’t suggest it but both sections are still focused on `console.log`.

Reply:
I see your point, and can think of 2 or 3 different ways to go on this. I'm going to make a change and see if we like it.  :)